navigator_api/main.py

```diff
@@ -70,58 +71,40 @@ def workflow_state(dataset_id):
         "id": "xxx-yyy-zzz",
         "milestones": [
             {
-                "id": "xxx",
-                "title": "Naomi Data Prep",
+                "id": mockup.id(),
+                "title": mockup.milestone(),
                 "completed": True,
                 "progress": 100
             },
             {
-                "id": "yyy",
-                "title": "Shiny 90 Data Prep",
+                "id": mockup.id(),
+                "title": mockup.milestone(),
                 "completed": False,
                 "progress": 50
             },
             {
-                "id": "zzz",
-                "title": "Update Spectrum",
+                "id": mockup.id(),
+                "title": mockup.milestone(),
                 "completed": False,
                 "progress": 0
             }
         ],
         "tasks": [
-            "aaabbbbcc",
-            "dddeeefff",
-            "asdfasdfa",
-            "aaabbbbcc",
-            "dddeeefff",
-            "asdfasdfa",
-            "aaabbbbcc",
-            "dddeeefff",
-            "asdfasdfa"
+            mockup.id(),
+            mockup.id(),
+            mockup.id(),
+            mockup.id(),
+            mockup.id(),
+            mockup.id(),
         ],
-        "currentTask": {
-            "id": "asdfasdfa",
-            "skipped": False,
+        "current_task": {
+            "id": mockup.id(),
+            "milestone_id": mockup.id(),
```

Comment:
adding milestone id in task content probably makes sense. We should talk about it with the team as it requires changes in engine docs.